tests/create_task_backend_parameters.yml

```diff
@@ -0,0 +1,45 @@
+name: Create Tes Task Job With Backend Parameters
+description: Job to create a new TES Task with backend parameters
+service: TES
+compatible_versions:
+  - v1.1
+tags:
+  - logical
+  - create task
+  - create task test
```

Comment:
Couldn't we use the task name as a test-specific "tag"? Makes no sense to add a(nother) test-specific to the tests...

Speaking of which (but a bit off topic): what happens if two tests with the same name are registered with the runner? Is there a defined behavior?

Reply:
I followed this structure for tags. 1st -> Logical/Functional. 2nd -> The primary endpoint dependent on the test. 3rd -> The individual test-specific name, in order to execute a single test. 4-> All. 

Currently, all tests have a unique test-specific tag. So, using it will only run that particular test as intended. However, if an "endpoint" is entered as tag. All tests related to that endpoint are run.

Reply:
I think you may be conflating your roles as test case developer and test runner developer a bit, leading to a bit of an abuse of tags, perhaps.

Tags are generally arbitrary key-values, it's difficult to enforce rules on those, and therefore it's dangerous to put any semantic assumptions on them. Things may work _now_ because you wrote both the runner code and the test cases and know all about them, but latest when someone else wants to use the runner for a different set of tests that you are not involved in, this will cause problems, for them and for you. See the next (very long) comment for a more detailed description of the problem(s) (it's a bit of a mess, sorry) and some suggestions on how to clean that up a bit.